app/controllers/application.js

```diff
@@ -62,6 +65,12 @@ export default Controller.extend(
 			});
 
 			this._super();
+
+			track({
+				action: trackActions.open,
+				category: 'connection-type',
+				label: `mobile-${this.get('connectionType')}`
+			});
```

Comment:
this event is nonInteractive
https://github.com/Wikia/mobile-wiki/blob/dev/app/utils/track.js#L127